### ISSUE TITLE
Document checklists for releasing / deploying / publishing

### ISSUE THREAD

**Issue by raoulmillais** [paritytech/substrate-connect#319]
## Intro

Ensure we have a thought out checklists (ideally as automated as possible).  This ticket is a placeholder, please flesh it out if you think of things and I'll ultimately split it up into work to actually be done.

## Processes to consider

* Publishing modules
* Deploying demos
* [Deploying code documentation (when publishing connect)](https://github.com/paritytech/substrate-connect/issues/220). Depends on #320 
* Deploying landing page
* Releasing the extension

## Things to think about

* Make documentation part of the contributing guidelines
* Release notes for @substrate/connect and the extension
* [Automate npm publishing](https://github.com/paritytech/substrate-connect/issues/304)

## Questions

* Where do we announce / who do we need to tell about releases & publishes?

**Comment by wirednkod** [comment]
We can use Releases/tags of github when releasing new version. Also probably a CHANGELOG.md will be good to have in the repo and keep it up to date.

**Comment by wirednkod** [comment]
Concerning the description of this issue:

- [x] Publishing modules
This is happening now with #1036 

- [x]  Deploying demos
This is happening now with #1036 

- [x] Deploying landing page
This is happening now with #1036 

- [ ] Releasing the extension
Script in #1037 is creating a zip upon build that can be uploaded to both chrome/firefox stores

**Comment by wirednkod** [comment]
Fixed with #1036 #1060
